iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/bicep/MemberExpressionImplTest.java

```diff
@@ -68,6 +68,10 @@ void shouldParseMemberExpression() {
       .matches("memberExpression!.functionCall()")
       .matches("memberExpression!::identifier123")
 
+      .matches("memberExpression.?identifier123")
+      .matches("memberExpression[?stringLiteral]")
+      .matches("memberExpression[?stringLiteral].?identifier123")
+
       .notMatches("memberExpression[stringLiteral")
       .notMatches("memberExpression!identifier123");
```

Comment:
Maybe also add a couple of negative examples with weird combinations of symbols like `:?` and `??`